src/test/java/org/wso2/carbon/connector/integration/test/nats/NatsConnectorIntegrationTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.connector.integration.test.nats;
+
+import org.json.JSONObject;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.connector.integration.test.base.ConnectorIntegrationTestBase;
+import org.wso2.connector.integration.test.base.RestResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Sample integration test
+ */
+public class NatsConnectorIntegrationTest extends ConnectorIntegrationTestBase {
+
+    private Map<String, String> eiRequestHeadersMap = new HashMap<String, String>();
+    private Map<String, String> apiRequestHeadersMap = new HashMap<String, String>();
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+
+        String connectorName = System.getProperty("connector_name") + "-connector-" +
+        System.getProperty("connector_version") + ".zip";
+        init(connectorName);
+
+        eiRequestHeadersMap.put("Accept-Charset", "UTF-8");
+        eiRequestHeadersMap.put("Content-Type", "application/json");
+    }
+
+    @Test(enabled = true, groups = {"wso2.ei"}, description = "nats test case")
+    public void testSample() throws Exception {
+        log.info("Successfully tested");
```

Comment:
At this point the request is not sent. Therefore can we say the successfully tested?